pliers/extractors/text.py

```diff
@@ -155,6 +163,20 @@ def _extract(self, stim):
                                self, features=['text_length'])
 
 
+class LowerCaseSpecialCharStripExtractor(TextExtractor):
+
+    ''' Strips the text of the special characters and converts to lower case. '''
+
+    VERSION = '1.0'
+
+    def _extract(self, stim):
+        
+        string=re.sub('[^ a-zA-Z0-9]', '', stim)
```

Comment:
This needs to be `stim.text`; the input `stim` is a `TextStim` instance, not a string. The text is stored in its `.text` attribute.